server-side-rendering/src/routes/signin/+page.server.ts

```diff
@@ -0,0 +1,28 @@
+import { SESSION_COOKIE, createAppwriteClient } from '$lib/server/appwrite.js';
+import { redirect } from '@sveltejs/kit';
+
+export function load({ locals }) {
+	if (locals.user) throw redirect(301, '/');
```

Comment:
Can we make those generic in layout?

For example:

Anything under `/auth` would redirect if already logged in, like `/auth/signin`, `/auth/signup`.
Anything under `/app` would redirect if not logged in, like `/app/dashboard`

Reply:
Sure 👍

Reply:
Don't see what value this adds tbh